Autocoders/Python/src/fprime_ac/generators/visitors/ComponentVisitorBase.py

```diff
@@ -442,6 +442,9 @@ def initPortFlags(self, obj, c):
         c.has_output_ports = len(c.output_ports) > 0
         c.has_typed_output_ports = len(c.typed_output_ports) > 0
         c.has_serial_output_ports = len(c.serial_output_ports) > 0
+        roles = [role for name, ptype, sync, priority, role, max_number in c.output_ports]
```

Comment:
Optional: use `_` for unused for loop variables.

Reply:
Using `_` here would be inconsistent with the rest of the file.  So I intentionally avoided that.

Reply:
:cry: I'm surprised LGTM isn't complaining about it.